angel_system/eval/gt_predict.py

```diff
@@ -0,0 +1,87 @@
+import time
+from typing import (
+    List,
+    Tuple,
+)
+
+import pandas
+import torch
+
+from angel_system.eval.support_functions import time_from_name
+
+
+# TODO: What is the best place for this function?
```

Comment:
I think this could be in `angel_system/utils/activity_classification.py` instead of here. I happen to interpret that this function is not assisting the evaluation code/tool in specific.

Reply:
Done